### PR TITLE
WIP Slice data science packages with count/offset

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -316,9 +339,17 @@ def cli(
...
             _LOGGER.info("dry-run: not scheduling data science packages solver jobs!")
+    else:
+        if data_science_packages_offset is not None:
+            _LOGGER.warning(
+                "Provided --data-science-packages-offset but solving data science packages will not be scheduled"
+            )
+        if data_science_packages_count is not None:
+            _LOGGER.warning(
+                "Provided --data-science-packages-offset but solving data science packages will not be scheduled"
+            )
```
Comment (harshad16) [inline app.py:352]: is this required, if the solve_data_science_package, is not set then, irrespective of the offset and count, packages will not be scheduled to solve, I guess one warning would be enough. right?

**Comment by fridex** [comment]
I'm thinking about usability here, maybe the approach is not correct. We will need some wait mechanism for this. Might be a good idea to incorporate such feature directly here. Marking as WIP.

**Comment by fridex** [comment]
No longer relevant. This will be handled on the message consumer side/investigator.

**Pull request closed without merging**